setup.py

```diff
@@ -39,6 +39,8 @@
         'cartopy>=0.18.0',
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
+        'emcpy @ git+https://github.com/NOAA-EMC/' +
```

Comment:
Is this cloning a specific hash? I wonder if we actually want develop?

Reply:
I feel like using develop can be dangerous, a specific hash ensures things don't break. But we can certainly switch to develop if you have a preference for that.

Reply:
Yeah I think as long as we don't let eva's hash of emcpy get too stale this should be the way to go. It ensures things do not break for users.